### DISCUSSION TITLE
GitHub Student Discount Not Working After Applying for 7 times in a row

### DISCUSSION THREAD

**Discussion by ghost** [community/community#52645] [GitHub Education]
### Select Topic Area

Question

### Body

Hello, I tried applying for GitHub Student several times. The first time i was told to use my webcam instead of a image. I did that but then it said that i am far away from my school. Then I saw that the device i was using had some location issue so i switched to a laptop and used a webcam to capture my student ID. I really do not like this. If you can't give students a little pass as they can't afford the subscription, why do you even have that at your webpage.

Image: -
![image](https://user-images.githubusercontent.com/102423760/231521386-631e5b2a-755e-4495-81c0-13eac7f61f55.png)

**Comment by jricciardi** [comment]
It looks like you marked this as resolved and closed it. If you found a solution or it resolved itself, please be sure to post a reply and mark the answer so others can find the resolution 😄 

Also, I'm moving this to the `Education` category, which is a more appropriate place for this.

**Comment by ghost** [comment]
I posted it in wrong category so that's why.

**Comment by ghost** [comment]
Hey can somone delete this discussion and [this one](https://github.com/orgs/community/discussions/52647); they are affecting privacy...
